client/app/api/course/index.js

```diff
@@ -22,6 +23,9 @@ const CourseAPI = {
   question: {
     scribing: ScribingQuestionAPI,
   },
+  answer: {
```

Comment:
Think this should be nested under AssessmentAPI, similar to: https://github.com/Coursemology/coursemology2/blob/master/client/app/api/course/Assessment/index.js#L8

Actually same with question, but we can fix that later.